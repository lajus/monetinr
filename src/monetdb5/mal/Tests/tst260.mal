# TopN queryies


b:= bat.new(:oid,:int);
err := bat.append(b,15);
err := bat.append(b,4);
n0  := calc.int(nil);
err := bat.append(b,n0);
err := bat.append(b,16);
err := bat.append(b,25);
err := bat.append(b,36);
err := bat.append(b,49);
err := bat.append(b,64);
err := bat.append(b,n0);
err := bat.append(b,n0);
err := bat.append(b,81);
io.print(b);
algebra.topN(b,9L);
io.print(b);
