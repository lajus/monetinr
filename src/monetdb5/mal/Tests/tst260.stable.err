stderr of test 'tst260` in directory 'monetdb5/mal` itself:


# 21:56:13 >  
# 21:56:13 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=34411 --set sql_port=49020 --set xquery_port=52098 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst260.mal < /dev/null
# 21:56:13 >  


# 21:56:13 >  
# 21:56:13 >  Done.
# 21:56:13 >  

