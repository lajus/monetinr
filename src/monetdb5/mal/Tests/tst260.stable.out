stdout of test 'tst260` in directory 'monetdb5/mal` itself:


# 21:56:13 >  
# 21:56:13 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=34411 --set sql_port=49020 --set xquery_port=52098 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst260.mal < /dev/null
# 21:56:13 >  

# Monet Database Server V4.99.19
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for x86_64-redhat-linux-gnu/64bit with 32bit OIDs; dynamically linked.
# config:/ufs/mk/monet5/src/mal/Tests/All.conf
# dbfarm:/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
## TopN queryies
#
#
#b:= bat.new(:int,:int);
#err := bat.insert(b,1,15);
#err := bat.insert(b,2,4);
#n0  := calc.int(nil);
#err := bat.insert(b,3,n0);
#err := bat.insert(b,4,16);
#err := bat.insert(b,5,25);
#err := bat.insert(b,6,36);
#err := bat.insert(b,7,49);
#err := bat.insert(b,8,64);
#err := bat.insert(b,9,n0);
#err := bat.insert(b,n0,n0);
#err := bat.insert(b,n0,81);
#io.print(b);
#algebra.topN(b,9L);
#io.print(b);
#
function user.main():void;
# TopN queryies 
    b := bat.new(:oid,:int);
    err := bat.append(b,15);
    err := bat.append(b,4);
    n0 := calc.int(nil);
    err := bat.append(b,n0);
    err := bat.append(b,16);
    err := bat.append(b,25);
    err := bat.append(b,36);
    err := bat.append(b,49);
    err := bat.append(b,64);
    err := bat.append(b,n0);
    err := bat.append(b,n0);
    err := bat.append(b,81);
    io.print(b);
    algebra.topN(b,9:lng);
    io.print(b);
end main;
#-----------------#
# h	t	  # name
# int	int	  # type
#-----------------#
[ 0@0,	  15	  ]
[ 1@0,	  4	  ]
[ 2@0,	  nil	  ]
[ 3@0,	  16	  ]
[ 4@0,	  25	  ]
[ 5@0,	  36	  ]
[ 6@0,	  49	  ]
[ 7@0,	  64	  ]
[ 8@0,	  nil	  ]
[ 9@0,	  nil	  ]
[ 10@0,	  81	  ]
#-----------------#
# h	t	  # name
# int	int	  # type
#-----------------#
[ 0@0,	  15	  ]
[ 1@0,	  4	  ]
[ 2@0,	  nil	  ]
[ 3@0,	  16	  ]
[ 4@0,	  25	  ]
[ 5@0,	  36	  ]
[ 6@0,	  49	  ]
[ 7@0,	  64	  ]
[ 8@0,	  nil	  ]

# 21:56:13 >  
# 21:56:13 >  Done.
# 21:56:13 >  

